TCS34725.h

```diff
@@ -60,7 +60,7 @@ class TCS34725_
     enum class Gain : uint8_t { X01, X04, X16, X60 };
 
     struct Color { float r, g, b; };
-    struct RawData { uint16_t r, g, b, c; };
+    struct __attribute__ ((packed)) RawData { uint16_t c, r, g, b; } ;
```

Comment:
I prefer using `union` instead of `struct` in this case

Reply:
I agree here because I learned something. :-)